styles/style.css

```diff
@@ -92,11 +103,41 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+.donate-btn {
+  background-color: #c05326;
+  color: #e4e4e4;
+  border: none;
+  padding: 0.6rem 0.8rem;
+  border-radius: 6px;
+  font-weight: 600;
+  transition: background-color ease-in-out 0.3s;
+  text-decoration: none;
+}
+
+.donate-btn:hover,
+.donate-btn:active {
+  background-color: white;
+  color: #c05326;
+}
 
+.volunteer-btn {
```

Comment:
Do you need to repeat all these styles again? What does class mean?

Can you make a class for all buttons, and then just set rules for the variations?